animations/test/misc/carousel_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/carousel.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createCarouselDemoScreen() => MaterialApp(
+      home: CarouselDemo(),
+    );
+
+void main() {
+  testWidgets('CarouselDemo swipe test', (tester) async {
```

Comment:
nit: This name isn't all that descriptive. Consider putting this in a group called "CarouselDemo tests" and then naming the test itself "Swipe left moves carousel."